.bazelrc

```diff
@@ -170,6 +170,7 @@ build --aspects //tools:aspects.bzl%mypy_aspect
 
 # optionally, default enable the mypy checks
 build --output_groups=+mypy
+build:local --output_groups=-mypy
```

Comment:
@RyanDraves, how do you run type-hinting because while mypy is technically working, it's super heavy weight to use with remote caching